Source/Core/Core/HW/Memmap.cpp

```diff
@@ -151,7 +153,10 @@ void MemoryManager::Init()
   m_physical_page_mappings_base = reinterpret_cast<u8*>(m_physical_page_mappings.data());
   m_logical_page_mappings_base = reinterpret_cast<u8*>(m_logical_page_mappings.data());
 
-  InitMMIO(wii);
```

Comment:
I don't know if all these changes are really gaining us much. The original call is clear enough, and it's used literally just once, and it would be immediately obvious if the `bool` was used incorrectly because the entire emulator would not work.